src/components/ResultItem/ResultItem.tsx

```diff
@@ -6,13 +6,13 @@ type meta = {
   lastUpdated?: string;
 };
 
-type ResultItemProps = {
+export interface ResultItemProps {
```

Comment:
Why `interface` and not `type`?

Reply:
What's the main difference between interface and type? I read few blogs and most of them uses interface and we are using type but not fully understand the differences.

Reply:
Re the difference this SO answer can tell you so much more about it
https://stackoverflow.com/questions/37233735/typescript-interfaces-vs-types#answer-52682220

I resolved to use type throughout as opposed to interface mainly because of point **5. Declaration merging** . If we allow multiple declarations it would make it much easier to accidentally break something.

In this instance I mistakenly thought I had to use interface to use a declaration in other files but that was incorrect (export is used to do that) so I've reverted to `export type ResultItemProps`

Reply:
Thanks for the link, really helpful